play/src/front/Components/EmbedScreens/CoWebsitesContainer.svelte

```diff
@@ -1,53 +1,392 @@
+            }
+            showArrow = true;
+        } else {
+            if (div) {
+                div.classList.add("hidden");
+            }
+            showArrow = false;
+        }
+    }
+
+    const setActiveCowebsite = (coWebsiteId: string) => {
+        activeCowebsite = coWebsiteId;
+        showDropdown = false;
+    };
+
+    const subscription = coWebsites.subscribe((arr) => {
+        activeCowebsite = arr[arr.length - 1]?.getId();
+    });
 
-<!-- Cette div devra apparaitre que s'il y a un event et elle peux apparaitre avec la methode display main dans le cowebsite manager-->
+    function toggleFullScreen() {
+        resizeBar = document.getElementById("resize-bar") as HTMLInputElement;
+        cowebsiteContainer = document.getElementById("cowebsites-container");
+
+        if (!document.fullscreenElement) {
+            if (cowebsiteContainer?.requestFullscreen) {
+                cowebsiteContainer.requestFullscreen().catch((e) => {
+                    console.error(e);
+                });
+                hideResizeBar();
+            }
+        } else if (document.exitFullscreen) {
+            document.exitFullscreen().catch((e) => {
+                console.error(e);
+            });
+            showResizeBar();
+        }
+    }
+
+    function hideResizeBar() {
+        const resizeBarInput = resizeBar as HTMLInputElement;
+        resizeBar.style.display = "none";
+        resizeBarInput.disabled = true;
+        resizeBar.style.cursor = "not-allowed";
+    }
+
+    function showResizeBar() {
+        const resizeBarInput = resizeBar as HTMLInputElement;
+        resizeBar.style.display = "block";
+        resizeBarInput.disabled = false;
+        resizeBar.style.cursor = "col-resize";
+    }
+
+    document.addEventListener("fullscreenchange", function () {
+        if (!document.fullscreenElement) {
+            showResizeBar();
+        }
+    });
+
+    function addDivForResize() {
+        const div = document.createElement("div");
+        div.id = "resize-overlay";
+        div.style.width = "100%";
+        div.style.height = "100%";
+        div.style.position = "absolute";
+        div.style.top = "0";
+        div.style.left = "0";
+        div.style.zIndex = "1000";
+        document.body.appendChild(div);
+
+        window.addEventListener("mouseup", removeDivForResize);
+    }
+
+    function removeDivForResize() {
+        const div = document.getElementById("resize-overlay");
+        if (div) {
+            document.body.removeChild(div);
+        }
+    }
+</script>
 
 <div
-    class="w-1/2 h-screen absolute right-0 top-0 bg-contrast/50 backdrop-blur z-[1000]"
-    class:vertical
-    transition:fly={{ duration: 750, x: 1000 }}
+    class={vertical
+        ? "w-1/2 h-full absolute right-0 top-0 bg-contrast/50 backdrop-blur z-[1500] left_panel responsive-container"
+        : "w-1/2 h-full absolute right-0 top-0 bg-contrast/50 backdrop-blur z-[1500] left_panel flex-col padding"}
```

Comment:
The `w-1/2` should be replaced to take into account the `getWidthPercent()` of the first cowebsite opened.